SecurityManager/Core.UnitTests/AclTools/Expansion/HtmlTagWriter/HtmlTagWriterTest.cs

```diff
@@ -117,25 +117,29 @@ public void SpecificTagsTest ()
 
 
     [Test]
-    [ExpectedException (typeof (XmlException), ExpectedMessage = "Wrong closing tag in HTML: Expected abc but was xyz.")]
     public void NonMatchingEndTagTest ()
     {
       using (var htmlWriter = new SecurityManager.AclTools.Expansion.HtmlTagWriter.HtmlTagWriter (TextWriter.Null, false))
       {
         htmlWriter.Tag("abc");
-        htmlWriter.TagEnd ("xyz");
+        Assert.That(

```

Comment:
Format

Reply:
What's wrong here?

Reply:
In 99% of the files changed there's a space between `.That` and `(`.

Reply:
*blink* Do'oh!